src/app/[locale]/seller/registration/page.tsx

```diff
@@ -194,7 +194,7 @@ const SellerRegistrationForm = () => {
       : UrlsRemoval(formData.sellerDescription);
 
     const formDataToSend = new FormData();
-    formDataToSend.append('name', formData.sellerName);
+    formDataToSend.append('name', UrlsRemoval(formData.sellerName));
```

Comment:
What happens if the name field is left empty?  I don't think we want to replace it with a [URL removed] placeholder.
To prevent this, you can use the same trim and clean logic as the address and description fields.

Reply:
If the name field is left blank, the default value will be an empty string. In the urls removal function, the regular expression will not find a match, allowing the form to submit with an empty string.

Reply:
You're absolutely right.  I realize now that I was a bit confused when I tested it locally.  Initially, the field had [URL Removed], and I changed the value to an empty string.  However, after saving, it reverted back to [URL Removed].  It seems that we're repopulating the previous value if the field is left blank. 🤔